br/pkg/lightning/mydump/csv_parser_test.go

```diff
@@ -838,13 +880,14 @@ func TestBackslashAsDelim(t *testing.T) {
 		CSV: config.CSVConfig{
 			Separator: ",",
 			Delimiter: `\`,
+			Null:      []string{""},
 		},
 	}
 
 	testCases := []testCase{
 		{
 			input:    `\\`,
-			expected: [][]types.Datum{{nullDatum}},
+			expected: [][]types.Datum{{types.NewStringDatum("")}},
```

Comment:
note that the behaviour is changed. empty field inside delimiter is no longer NULL

Reply:
Oh I just noticed we have explicitly stated the behaviour in doc

> Quoting does not affect whether a field is null.

https://docs.pingcap.com/tidb/stable/tidb-lightning-data-source#not-null-and-null

I should add a hide configuration for LOAD DATA 😂